git-lfs/git-lfs.install

```diff
@@ -0,0 +1,21 @@
+post_install() {
+  for arch in mingw32 mingw64; do
+    if [ -f "${arch}/bin/git-lfs.exe" ]; then
+      git config --system filter.lfs.clean "git-lfs clean -- %f"

```

Comment:
Let's replace the `--system` with `-f "${arch}/etc/gitconfig"`. That way, we can be certain that the *correct* system gitconfig gets updated.